src/components/Player/Player.tsx

```diff
@@ -56,14 +68,17 @@ export default function Player(props: IPlayerProps): JSX.Element {
         />
       </div>
       {names && (
-        <div className={classes.name}>
-          <Input
+        <div className={playerStyles.name}>
+          <input
+            type="text"
             placeholder="Player"
-            className={classes.input}
+            className={`${playerStyles.input} ${inputStyles.root}`}
```

Comment:
There seems to be a problem with the text position, is center... I think you can just remove the inputStyles and style it using the components styles, we still do not know if the header will be removed or not.


PS: if you plan on contributing a bit more on this project, we will be having a small meeting about what we will focus on next on discord, add me if interested: SykO#9032